script.js

```diff
@@ -0,0 +1,908 @@
+  payOutScheduleDisplay.appendChild(nameOfCombiDisplay);
+
+  // Create the display for payoutSchedule
+  for (let i = 0; i < payOutSchedule.length; i += 1) {
+    const payOutColumn = document.createElement('div');
+    payOutColumn.classList.add(`pCol${i + 1}`);
+    for (let j = 0; j < payOutSchedule[i].length; j += 1) {
+      const payOutBox = document.createElement('div');
+      payOutBox.classList.add('pOutBox');
+      payOutBox.innerText += payOutSchedule[i][j] + '\t';
+      payOutColumn.appendChild(payOutBox);
+    }
+    payOutScheduleDisplay.appendChild(payOutColumn);
+  }
+};
+
+// Function that creates a button that allows users to increment credits
+const createInsertCreditsBtn = () => {
+  const insertCreditsBtn = document.createElement('button');
+  insertCreditsBtn.classList.add('btn');
+  insertCreditsBtn.classList.add('btn-design1');
+  insertCreditsBtn.innerText = 'INSERT \n 1 CREDIT';
+  insertCreditsBtn.setAttribute('id', 'insertCreditsBtn');
+  insertCreditsBtn.addEventListener('click', () => {
+    if (numCreditsInserted < 5) {
+      numCreditsInserted += 1;
+      creditsInsertedDisplay.innerText = `Insert Credit(s): ${numCreditsInserted}`;
+      const leftPxArray = ['490px', '598px', '707px', '816px', '925px'];
+      switch (numCreditsInserted) {
+        case 1:
+          payOutLevelPointer.style.left = `${leftPxArray[0]}`;
+          break;
+        case 2:
+          payOutLevelPointer.style.left = `${leftPxArray[1]}`;
+          break;
+        case 3:
+          payOutLevelPointer.style.left = `${leftPxArray[2]}`;
+          break;
+        case 4:
+          payOutLevelPointer.style.left = `${leftPxArray[3]}`;
+          break;
+        case 5:
+          payOutLevelPointer.style.left = `${leftPxArray[4]}`;
+          break;
+        default:
+          payOutLevelPointer.style.left = `${leftPxArray[0]}`;
+      }
+
+      console.log(numCreditsInserted, 'creditsToBeInserted');
+    }
+  });
+  buttonsContainer.appendChild(insertCreditsBtn);
+};
+// Helper Function that uses runDealCardsEngine and game state to output relevant msgs
+const dealCardsAndAnimateMsg = () => {
+  // Animation for output message in stats display when moving on to next round
+  // When player begins gameRound 1
+  if (gameState === 'startGame') {
+    console.log(gameState);
+    runDealCardsEngine();
+    gameState = 'playerNextMove';
+    // When player decides to either 'deal' or 'swap'
+  } else if (gameState === 'playerNextMove' || gameState === 'gameOver') {
+    calcHandScore();
+    createGameOverDisplay();
+    statsDisplay.innerText = '';
+    // Empty currAmtWon to total credits left
+    creditsLeft += currAmtWon;
+    createGameStatsDisplay('Dealing...');
+    if (currAmtWon > 0) {
+      creditsLeftDisplay.classList.add('animate__animated');
+      creditsLeftDisplay.classList.add('animate__pulse');
+      creditsLeftDisplay.classList.add('animate__fast');
+      creditsLeftDisplay.classList.add('animate__repeat-3');
```

Comment:
Details of animation can be separated into helper function to make the higher-level logic clearer in this function.